zulipterminal/ui_tools/boxes.py

```diff
@@ -366,30 +365,19 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         if not reactions:
             return ''
         try:
```

Comment:
Perhaps as a follow-up commit we could narrow down what this `try` block is intended to do? Not everything needs to be in it, and wrapping everything makes it less clear.